infra/resources/_modules/function_apps_msgs_app/locals.tf

```diff
@@ -0,0 +1,49 @@
+locals {
+  messages_app = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME       = "node"
+      WEBSITE_RUN_FROM_PACKAGE       = "1"
+      WEBSITE_DNS_SERVER             = "168.63.129.16"
+      FUNCTIONS_WORKER_PROCESS_COUNT = 4
```

Comment:
Any reason to have this one?

Reply:
Coherence with other modules. I'll investigate if it's still useful.

Reply:
FUNCTIONS_WORKER_PROCESS_COUNT this is still usefull https://learn.microsoft.com/en-us/azure/azure-functions/functions-app-settings#functions_worker_process_count